infra/growth/src/util/arrayUtils.js

```diff
@@ -0,0 +1,9 @@
+function concat(x, y) {
+  return x.concat(y)
+}
+
+function flatMap(f, xs) {
+  return xs.map(f).reduce(concat, [])
+}
+
+module.exports = { concat, flatMap }
```

Comment:
nit: no need to export concat.

Reply:
tnx, incorporating @shahthepro's change the concat is removed